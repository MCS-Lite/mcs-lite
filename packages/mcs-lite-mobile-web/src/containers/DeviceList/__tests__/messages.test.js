it('should return messages', () => {
  expect(require('../messages').default).toMatchSnapshot();
});
