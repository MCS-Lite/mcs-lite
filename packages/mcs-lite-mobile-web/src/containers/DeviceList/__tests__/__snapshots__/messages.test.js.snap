exports[`test should return messages 1`] = `
Object {
  "DeviceList.myTestDevices": Object {
    "defaultMessage": "我的測試裝置",
    "description": "Title",
    "id": "DeviceList.myTestDevices",
  },
  "DeviceList.noDevice": Object {
    "defaultMessage": "目前還沒有任何測試裝置",
    "id": "DeviceList.noDevice",
  },
  "DeviceList.search": Object {
    "defaultMessage": "搜尋",
    "id": "DeviceList.search",
  },
}
`;
